src/main/scala/sigmastate/utils/SigmaByteReader.scala

```diff
@@ -2,10 +2,16 @@ package sigmastate.utils
 
 import scorex.util.Extensions._
 import scorex.util.serialization.Reader
+import java.nio.ByteBuffer
+
+import org.ergoplatform.validation.SigmaValidationSettings
+import scorex.util.serialization.{VLQByteBufferReader, Reader}
 import sigmastate.SType
 import sigmastate.Values.SValue
 import sigmastate.lang.exceptions.DeserializeCallDepthExceeded
-import sigmastate.serialization.{ConstantStore, SigmaSerializer, TypeSerializer, ValDefTypeStore, ValueSerializer}
+import sigmastate.serialization.{ConstantStore, TypeSerializer, ValDefTypeStore, ValueSerializer, SigmaSerializer}
+import sigmastate.serialization.{ValDefTypeStore, TypeSerializer, ValueSerializer, ConstantStore}
+import scorex.util.Extensions._
```

Comment:
This import is already defined above. Please remove unused imports

Reply:
done